config.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Configuration for database connection
+ *
+ */
+ //Not sure if these fields are right
+$host       = "csdb.brockport.edu";
+$username   = "kdarl1";
+$password   = "webdesign";
+$dbname     = "test";
+$dsn        = "mysql:host=$host;dbname=$dbname";
+$options    = array(
+                PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION
+              );
```

Comment:
This file is used as a config. Sets variables for each bit of information needed to connect to database, i.e. host, name, password, etc. I'm unsure if the information I have is correct or if we need to put something different in.